scripts/create-clusters.sh

```diff
@@ -75,16 +75,17 @@ function configure-insecure-registry-and-reload() {
 }
 
 function insecure-registry-config-cmd() {
-  if test -f "${docker_daemon_config}"; then
-    sed -i "1a \    \"insecure-registries\": [\"${CONTAINER_REGISTRY_HOST}\"]" ${docker_daemon_config}
-  else
-    echo "cat <<EOF > ${docker_daemon_config}
+  echo "
```

Comment:
What is your intent in echoing a command instead of executing it?

Reply:
The returned value of this function `insecure-registry-config-cmd` is to be executed as a command. I just overlooked it before. 
```
${cmd_context} "$(insecure-registry-config-cmd)"
```